### ISSUE TITLE
(AreaChart) Allow overflow of data points in non-scrollable chart

### ISSUE THREAD

**Issue by witaylor** [Abhinandan-Kushwaha/react-native-gifted-charts#623]
When creating an AreaChart, the last data point gets clipped. For my use case, the gradient under the graph is fine but I would like the full circle of the data point to show. Is there a way to do this? 

## Screenshot

<img width="342" alt="Screenshot 2024-05-05 at 19 11 31" src="https://github.com/Abhinandan-Kushwaha/react-native-gifted-charts/assets/41151984/c5c37ebe-c0b0-4a40-a376-cadf2e616c0a">

## Code example

**Creating the data:**

```typescript
const points = [66, 50, 48, 60, 75, 70, 72].map((value, idx, arr) => ({
  value,
  customDataPoint: () =>
    // only show point on the last entry
    idx === arr.length - 1 && (
      <View
        style={{
          width: 20,
          height: 20,
          backgroundColor: "white",
          borderWidth: 4,
          borderRadius: 10,
          borderColor: "#07BAD1",
        }}
      />
    ),
}));
```

**The component:**

```typescript
<LineChart
  data={points}
  width={325}
  xAxisThickness={0}
  yAxisThickness={0}
  initialSpacing={0}
  adjustToWidth
  thickness={3}
  hideRules
  stepValue={20}
  maxValue={90}
  interpolateMissingValues
  color="#07BAD1"
  curved
  curveType={1}
  areaChart
  startFillColor="#07BAD1"
  startOpacity={0.8}
  endFillColor="#07BAD1"
  endOpacity={0.3}
  isAnimated
  disableScroll
  pointerConfig={{}}
/>
```

**Comment by julienclaerhout** [comment]
Certainly due to the 'adjustToWidth' props.
Try spacing={} props or reduce yAxisLabels width

**Comment by erdaulhaq** [comment]
i have save problem

**Comment by Abhinandan-Kushwaha** [comment]
Hi @witaylor @julienclaerhout @erdaulhaq 👋
Thanks for reporting this issue. It is fixed in versions 1.4.13 onwards. 🎉

Now we are also adding the `dataPointsRadius` to the width of the chart, so the data point will not be cropped.
Please use the latest version of the library.

**Comment by MartinLoidolt** [comment]
Hi everyone,

I am having a similar/same problem.
But I am using a custom PointerComponent.

Does anyone know why this is happening?

![Image](https://github.com/user-attachments/assets/6f82c34e-ce1e-428e-b327-2d350b27bf56)
